sdk/iot/hub/inc/az_iot_hub_client.h

```diff
@@ -411,7 +416,7 @@ typedef struct az_iot_hub_client_method_request
  * @param[out] out_request If the message is a method request, this will contain the
  *                         #az_iot_hub_client_method_request.
  * @return #az_result
- *         - #AZ_ERROR_IOT_TOPIC_NO_MATCH if the topic is not matching the expected format.
+ *         - `AZ_ERROR_IOT_TOPIC_NO_MATCH` if the topic is not matching the expected format.

```

Comment:
Same here/elsewhere related to `az_result`. Keep `#`.

Reply:
See above.